sdk/storage/azblob/.gitignore

```diff
@@ -0,0 +1,298 @@
+## Ignore generated bin files
+BigFile*.bin
```

Comment:
You probably don't need this file

Reply:
Yes lets try to avoid gitignore files that aren't in the root.

Reply:
Our unit test creates this as a part of testing upload/download of file through SDK. We've added it to make sure we don't accidentally commit it in the main branch. 
I think you may be confusing it with some other standard file in GO SDK.

Reply:
The SDK repo has a .gitignore file at the root of the repo.  We avoid having .gitignore files other than the one at the root.
If you have files that need to be excluded the .gitignore at the root should be updated.

Reply:
Thanks for letting me know. I'll do it.